src/app/dataValidator/rules/topstories.js

```diff
@@ -0,0 +1,55 @@
+import deepClone from '../../helpers/json/deepClone';
+import deepGet from '../../helpers/json/deepGet';
+
+const squashKeys = [
+  'responsive-top-stories',
+  'top-story',
+  'secondary-top-story',
+  'other-top-stories',
+];
+
+const squashTopStories = jsonRaw => {
+  const json = deepClone(jsonRaw); // make a copy so we don't corrupt the input
+  let groups = deepGet(['content', 'groups'], json);
+  let collectedItems = [];
+
+  if (groups) {
+    /*
+     * Find and delete unwanted groups while collecting items
+     */
+    groups = groups.filter(group => {
+      if (group.type && squashKeys.includes(group.type)) {
```

Comment:
```suggestion
      if (squashKeys.includes(group.type)) {
```

I don't think `group.type` is needed here because `squashKeys.includes(undefined)` returns `false`. If you were trying to guard against `group` being `undefined` then this also isn't achieving that because both sides of the `&&` are always evaluated even if the first fails. 

![image](https://user-images.githubusercontent.com/7791726/56302189-2d21ea80-6131-11e9-939b-4a4c65676954.png)

Reply:
Done :)